Makefile

```diff
@@ -29,7 +29,8 @@ BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 #
 # For example, running 'make bundle-build bundle-push catalog-build catalog-push' will build and push both
 # kiegroup.org/kogito-serverless-operator-bundle:$VERSION and kiegroup.org/kogito-serverless-operator-catalog:$VERSION.
-IMAGE_TAG_BASE ?= quay.io/dsalerno/kogito-serverless-operator
+IMAGE_TAG_BASE ?= quay.io/mdessi/kogito-serverless-operator
```

Comment:
I think that it is better to revert this change so that when we will move the project to kiegroup org the replace will be easier.

Reply:
ok